Automattic/jetpack
# CLI: Add Jetpack Test Command

Fixes #20761

#### Changes proposed in this Pull Request:

* Adds a `jetpack test` command that lets you run scripts defined in the project's composer.json files. 

Surely missing a few use cases here, but hopefully this is a good start to make running tests easier.

#### Jetpack product discussion



#20761

#### Does this pull request change what data or activity we track or use?



No

#### Testing instructions:






* Pull down this PR and run `jetpack test`
* Follow the prompts to run the test you want.
* Accepts two parameters:
	* `project` in the form of `project/name`, ie `plugins/jetpack`
	* `test,` accepting `js`, `php`, or `coverage`
* ie, `jetpack test plugins/jetpack js` should run JavaScript tests.
